eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala

```diff
@@ -198,6 +205,34 @@ object EventualCassandra {
             } else {
               events
             }
+
+            if (dataIntegrity.correlateEventsWithMeta) {
```

Comment:
I don't think that we actually need "read-only" mode because:

1. its pretty rare case and I cannot predict for how long "read-only" mode should be tested
2. _if_ `seqNr` are duplicated then another `Data integrity violated` error will be thrown